client/blocks/product-purchase-features/product-purchase-features-list/video-audio-posts.jsx

```diff
@@ -12,17 +12,32 @@ import { localize } from 'i18n-calypso';
  */
 import PurchaseDetail from 'components/purchase-detail';
 import paths from 'lib/paths';
+import { PLAN_BUSINESS } from 'lib/plans/constants';
+
+export default localize( ( { selectedSite, plan, translate } ) => {
+	let featureDescription;
+
+	switch ( plan ) {
```

Comment:
I think it would be better if we explicitly list the possible plans instead of making a case just for Business and having Premium as default. Imagine somebody used this component for the Personal plan, totally forgetting about writing description for it. The Premium's one would be used.

`default` can be an empty text in that case. I think it's easier to notice blank description than to notice it's a description for Premium plan.

Reply:
Sounds good! Updated to an empty string for `default`.